src/apps/api/serializers/submissions.py

```diff
@@ -41,6 +41,7 @@ class SubmissionSerializer(serializers.ModelSerializer):
     on_leaderboard = serializers.BooleanField(read_only=True)
     task = TaskSerializer()
     created_when = serializers.DateTimeField(format="%Y-%m-%d %H:%M")
+    auto_run = serializers.SerializerMethodField(read_only=True)
```

Comment:
Why do we need to add `auto_run` to every submission? To me if you want to see if a submission should be auto run, you should check the associated competition?

Reply:
This is added so that it can be accessed easily because submission is not directly linked to a competition but through a phase